examples/mouse_position.rs

```diff
@@ -0,0 +1,84 @@
+use bevy::{input::InputSystem, prelude::*, window::PrimaryWindow};
+use leafwing_input_manager::{
+    axislike::DualAxisData, plugin::InputManagerSystem, prelude::*, systems::run_if_enabled,
+};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_plugin(InputManagerPlugin::<BoxMovement>::default())
+        .add_startup_system(setup)
+        .add_system(
+            update_cursor_state_from_window::<BoxMovement>
+                .run_if(run_if_enabled::<BoxMovement>)
+                .in_base_set(CoreSet::PreUpdate)
+                .in_set(InputManagerSystem::ManualControl)
+                .before(InputManagerSystem::ReleaseOnDisable)
+                .after(InputManagerSystem::Tick)
+                .after(InputManagerSystem::Update)
+                .after(InputSystem),
+        )
+        .add_system(pan_camera)
+        .run();
+}
+
+#[derive(Actionlike, Clone, Debug, Copy, PartialEq, Eq)]
+enum BoxMovement {
+    Pan,
+}
+
+fn setup(mut commands: Commands, window: Query<Entity, With<PrimaryWindow>>) {
+    commands.spawn(Camera2dBundle::default());
+
+    let entity = commands
+        .spawn(SpriteBundle {
+            transform: Transform::from_scale(Vec3::new(100., 100., 1.)),
+            ..default()
+        })
+        .insert(InputManagerBundle::<BoxMovement>::default())
+        // Note: another entity will be driving this input
+        .id();
+
+    commands.entity(window.single()).insert(ActionStateDriver {
+        action: BoxMovement::Pan,
+        targets: entity.into(),
+    });
+}
+
+fn update_cursor_state_from_window<A: Actionlike>(
```

Comment:
Hmm, it's nice to demonstrate the power of generics but a bit weird when we're only using a single type in the example.

Maybe a comment here calling out what `A` is?

Reply:
I just copied and pasted the ui signature, going to just remove the generic here.